src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/on_system_request_notification_test.cc

```diff
@@ -160,6 +175,57 @@ TEST_F(OnSystemRequestNotificationTest, Run_HTTPType_SUCCESS) {
             (*msg)[strings::params][strings::protocol_version].asInt());
 }
 
```

Comment:
Stop review

Reply:
@SNiukalov What does it mean `Stop review` ?